test/spec_response.rb

```diff
@@ -408,7 +408,7 @@ def object_with_each.each
     status.must_equal 404
   end
 
-  it "correctly updates content-type when writing when not initialized with body" do
+  it "correctly updates content-length when writing when initialized without body" do
```

Comment:
🗒️ These specs incorrectly referred to `content-type`, but are actually testing `content-length`.